help/print-a-topic.md

```diff
@@ -0,0 +1,12 @@
+# Print a topic
+
+Topics are printable in Zulip. To save paper, printed topics exclude sidebars
```

Comment:
Are specific direct message conversations not printable? I would assume they might be because they are similar views.

Also, I would delete "To save paper" and start the sentence with "Printed ..."